src/gui/basedataviewmodel.h

```diff
@@ -7,6 +7,8 @@
 #include <climits>
 #include <set>
 #include "log.h"
+#include "../utils/conversion.h"
```

Comment:
why is this include needed? additional includes in header files should be avoided when possible.